src/Unleash/Variants/VariantPayload.cs

```diff
@@ -1,31 +1,9 @@
-using System;
-
 namespace Unleash.Variants
 {
-    public class Payload
+    public class Payload: Yggdrasil.Payload
```

Comment:
Just wondering again why we have to keep this one around, I can't remember the exact reason

Reply:
Same as https://github.com/Unleash/unleash-client-dotnet/pull/224/files#r1661227880 perhaps?

Reply:
yup